src/components/BoardingForm.js

```diff
@@ -1,10 +1,24 @@
-import React, { useState } from "react";
-import { Form, Wrapper } from "../styling/StylingCheckInPage";
+import React, { useEffect, useState } from "react";
+import {
+  Form,
+  Wrapper,
+  WrapperTime,
+  TimeSection,
+  ButtonSection,
+} from "../styling/StylingCheckInPage";
 import Button from "./Button";
-import Alert from "./Alert";
 import ListBoarding from "./ListBoarding";
 import { nanoid } from "nanoid";
 
+const getLocalStorage = () => {
+  let listBoarding = localStorage.getItem("listBoarding");
+  if (listBoarding) {
+    return JSON.parse(localStorage.getItem("listBoarding"));
```

Comment:
Hier wäre es besser den schon gelesenen Inhalt (`listBoarding`) zu parsen, statt nochmal neu aus dem LocalStorage zu lesen. Das reduziert den duplizierten Code und gibt Sicherheit, dass der Inhalt des LocalStorage nicht zwischenzeitlich geleert wurde.

```suggestion
    return JSON.parse(listBoarding);
```